lib/releaselib.sh

```diff
@@ -1198,9 +1193,16 @@ release::set_globals () {
     BUCKET_TYPE="release"
   fi
 
+  # The "production" GCR path is now multi-region alias
+  GCRIO_PATH_PROD="k8s.gcr.io"
+  # The "production" push path to that multi-region alias has a staging- prefix
+  GCRIO_PATH_PROD_PUSH="staging-$GCRIO_PATH_PROD"
```

Comment:
it doesn't look like this is used anywhere?

Reply:
It is now.  Please check the use of it in the new release::docker::release().